Composer/packages/client/src/pages/design/PropertyEditor.tsx

```diff
@@ -81,8 +82,7 @@ const PropertyEditor: React.FC = () => {
   }, [$schema, formUIOptions]);
 
   const errors = useMemo(() => {
-    const diagnostics = currentDialog?.diagnostics;
-    if (diagnostics) {
+    if (diagnostics && diagnostics.length) {
```

Comment:
```suggestion
    if (diagnostics?.length) {
```

Reply:
This seems like something the linter could catch. I'll investigate.

Reply:
Found it. @GeoffCoxMSFT or @hatpick can you create a PR to update the lint rules with this?

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-optional-chain.md

Reply:
I can do that.